pkg/sql/parser/testdata/alter_default_privileges

```diff
@@ -62,6 +62,15 @@ ALTER DEFAULT PRIVILEGES GRANT ALL ON FUNCTIONS TO foo, bar -- fully parenthesiz
 ALTER DEFAULT PRIVILEGES GRANT ALL ON FUNCTIONS TO foo, bar -- literals removed
 ALTER DEFAULT PRIVILEGES GRANT ALL ON FUNCTIONS TO _, _ -- identifiers removed
 
+error
+ALTER DEFAULT PRIVILEGES GRANT ALL ON PROCEDURES TO foo
```

Comment:
Is this being implemented in another PR? Or do PROCEDURES not have default privileges?

Reply:
It's a syntax error in Postgres:

```
marcus=# ALTER DEFAULT PRIVILEGES GRANT ALL ON PROCEDURES TO foo;
ERROR:  42601: syntax error at or near "PROCEDURES"
LINE 1: ALTER DEFAULT PRIVILEGES GRANT ALL ON PROCEDURES TO foo;
```

And this decision is backed-up by the [Postgres docs](https://www.postgresql.org/docs/current/sql-alterdefaultprivileges.html):

> For this command, functions include aggregates and procedures. The words FUNCTIONS and ROUTINES are equivalent in this command. (ROUTINES is preferred going forward as the standard term for functions and procedures taken together. In earlier PostgreSQL releases, only the word FUNCTIONS was allowed. It is not possible to set default privileges for functions and procedures separately.)

Reply:
Ah ok, good to know.

Do we normally return a syntax error for situations like this? It is not very user friendly IMO since it does not convey that "It is not possible to set default privileges for functions and procedures separately."

Reply:
My aim was to mimic Postgres, but a more user-friendly error could be nice. Feel free to create an issue and we can put it on the backlog.